jestconfig.json

```diff
@@ -6,6 +6,7 @@
     "./node_modules/enzyme-to-json/serializer"
   ],
   "moduleNameMapper": {
-    "\\.(css|scss|svg)$": "identity-obj-proxy"
+    "\\.(svg)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
Where are you using this file? I don't see a reference in the tests.

Reply:
it is necessary for Jest to properly import svg files. [ref](https://facebook.github.io/jest/docs/en/webpack.html#handling-static-assets)

Reply:
> necessary for Jest to properly handle imports of svg files

Ah this statement here answers my question. I was wondering where it was getting imported at to be used in the tests. I realize it his here.

Give this, I feel we should Map fileMock to it's path, not every .svg file.

Reply:
Every svg (or any other static assets for that matter) that is used in Jest has to be mocked or Jest will fail to execute. Since the scope of this mock is Jest testing, I'm not sure providing a specific path is advantageous here.

Or am I not understanding correctly what you are recommending? Can you elaborate?

Reply:
From what I am understanding, every svg will be mapped to fileMock.js based on this config
``` "\\.(svg)$": "<rootDir>/__mocks__/fileMock.js",```

I was wondering if we should update this to use regex to specify which fileMock. I have been thinking on it though, and if we run into an instance were we would expect a different mock we could update this then.